### ISSUE TITLE
[conan.io/center] parallel-hashmap/1.31 merged but not found in conan center

### ISSUE THREAD

**Issue by SpaceIm** [conan-io/conan-center-index#1475]
Even though https://github.com/conan-io/conan-center-index/pull/1253 has been merged, `parallel-hashmap/1.31` can't be found in Web UI or with `conan search`

**Comment by solvingj** [comment]
Indeed, there's something wrong. I will work on this.

```
C:\Users\jerryw\source\git-conan\bzip2_license\recipes\bzip2\1.0.8>conan search parallel-hashmap/1.30 -r conan-center
Existing package recipes:

parallel-hashmap/1.30

C:\Users\jerryw\source\git-conan\bzip2_license\recipes\bzip2\1.0.8>conan search parallel-hashmap/1.31 -r conan-center
There are no packages matching the 'parallel-hashmap/1.31' pattern

```
And this page says 1.30 is the only available version: 

![image](https://user-images.githubusercontent.com/8557737/80770529-88493400-8b1e-11ea-869d-00cb30ed3a79.png)

Yet... this URL does work: 

https://conan.io/center/parallel-hashmap/1.31/?user=_&channel=_

For better or worse, that page agree's that the only available version is 1.30

![image](https://user-images.githubusercontent.com/8557737/80770434-51731e00-8b1e-11ea-8413-84ed9dae1d74.png)

**Comment by danimtb** [comment]
I have created a new PR for parallel-hashmap in order to regenerate the packages here: https://github.com/conan-io/conan-center-index/pull/1534

Apparently the GitHub webhook failed to notify our CI Jenkins about the merge, and the job was not created and packages were never uploaded.

We are planning to add an internal job that checks for missing packages in ConanCenter in order to minimize this for the future.

Thanks a lot for reporting @SpaceIm
